### DISCUSSION TITLE
Add label "com.docker.compose.project" to all containers created by the Maincontainer

### DISCUSSION THREAD

**Discussion by KaeTuuN** [nextcloud/all-in-one#3950] [Ideas] [answered]
As an Admin it would be great to see, if all containers created by the "nextcloud-aio-mastercontainer" would have the same value for "com.docker.compose.project" as the Mastercontainer.

This is especially helpful to sort your environments, i.e. in Portainer all Container with the same "com.docker.compose.project" will appear under the same Stack.
Another usecase is the automatic assigning of userrights to manage container.

**Comment by szaimen** [comment]
Hi, currently there are no plans to implement this as the use case is very limited. However you might make it work with https://github.com/nextcloud/all-in-one/tree/main/manual-install

**Comment by szaimen** [comment, marked as answer]
Hi, this is now getting worked on in https://github.com/nextcloud/all-in-one/pull/7190

**Reply by szaimen** [reply]
This is now released with v12.2.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
